spec/controllers/forgot_passwords_controller_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'spec_helper'
+
+describe ForgotPasswordsController do
+  describe "POST create" do
+    context "with blank input" do
+      it "redirects to the fogot password page" do
+        post :create, email: ""
+        expect(response).to redirect_to forgot_password_path
+      end
+      
+      it "shows an error" do
+        post :create, email: ""
+        expect(flash[:error]).to eq("Email can not be blank.")
```

Comment:
You can also just test that `flash[:error]` is present, instead of coupling your spec to the specific string that is in your implementation.